packages/esm-patient-common-lib/src/dashboards/DashboardExtension.tsx

```diff
@@ -1,15 +1,23 @@
-import React from 'react';
+import React, { useMemo } from 'react';
+import { last } from 'lodash';
+
 import { ConfigurableLink } from '@openmrs/esm-framework';
+import styles from './dashboardextension.scss';
 
 export interface DashboardExtensionProps {
   title: string;
   basePath: string;
+  currentPath: string;
 }
 
-export const DashboardExtension = ({ title, basePath }: DashboardExtensionProps) => {
+export const DashboardExtension = ({ title, basePath, currentPath }: DashboardExtensionProps) => {
+  const navLink = useMemo(() => decodeURIComponent(last(currentPath.split('/'))), [currentPath]);
+
+  const activeClassName = title === navLink ? 'active-left-nav-link' : 'non-active';
+
   return (
-    <div key={title}>
-      <ConfigurableLink to={`${basePath}/${encodeURIComponent(title)}`} className="bx--side-nav__link">
+    <div key={title} className={activeClassName}>
+      <ConfigurableLink to={`${basePath}/${encodeURIComponent(title)}`} className={'bx--side-nav__link ' + styles.link}>
```

Comment:
I think I'm missing something? Currently that styles file doesn't have a `link` class name?

Reply:
It's there in the styles maybe you had not refreshed.